go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -11367,16 +11367,17 @@ func TestValidateVersionShard(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		curT := tt
 		t.Run(tt.name, func(t *testing.T) {
-			tt.setup()
-			resp, err := vtctld.ValidateVersionShard(ctx, tt.req)
-			if tt.shouldErr {
+			curT.setup()
+			resp, err := vtctld.ValidateVersionShard(ctx, curT.req)
+			if curT.shouldErr {
```

Comment:
@ajm188 main and release-16.0 have diverged. release-16.0 still has `t.Parallel()`. Is this change really required now that we no longer `t.Parallel()` in this test?

Reply:
I removed this change in 425362d26b and `golangci-lint` did not complain locally. Closing this for now.